daml-foundations/daml-ghc/daml-prim-src/GHC/Num.daml

```diff
@@ -33,23 +33,23 @@ infixr 8 ^
 infixl 7 /
 infixl 7 %
 
--- | Basic additive class.
--- Instances are expected to respect the following laws:
+-- | Use the `Additive` class for types that can be added.
+-- Instances have to follow these rules:
```

Comment:
I think "respect the following laws" is a better formulation. After all, they are called "laws" in basic arithmetic. Same for multiplicative below.